substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -329,12 +333,10 @@ impl PalletCmd {
 		let mut component_ranges = HashMap::<(Vec<u8>, Vec<u8>), Vec<ComponentRange>>::new();
 		let pov_modes = Self::parse_pov_modes(&benchmarks_to_run)?;
 
-		for (pallet, extrinsic, components, _) in benchmarks_to_run.clone() {
+		for (pallet, extrinsic, components, _, pallet_str, extrinsic_str) in benchmarks_to_run.clone() {
```

Comment:
```suggestion
		for (pallet, extrinsic, components, _, pallet, extrinsic) in benchmarks_to_run.clone() {
```
Types dont need to be in the var name.